RNW v0.75.4 & RN-WINRT v0.72.1 No Longer Builds

### The Problem:
Originally, I had a solution with this exact setup that was working. I then removed some bloat from Visual Studio 2019 and 2022 that as far as I know should not affect react-native-winrt. I have the required dependencies outlined on the docs with the addition of 10.0.18362 through visual studio 2022 as requested by react-native-winrt. 


### Steps to Reproduce:
`yarn add react-native-windows@^0.75.4`
`yarn react-native init-windows --overwrite`
`yarn react-native run-windows`
Builds with no issue.
Stop debug session.
`yarn add react-native-winrt`
`yarn react-native run-windows`

### Error
`C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\XamlCompiler\Microsoft.Windows.UI.Xaml.Common.targets(482,5): error MSB4181: The "CompileXaml" task returned false but did not log an error. [C:\Users\source\repos\projects\ExampleApp\node_modules\react-native-winrt\windows\WinRTTurboModule\WinRTTurboModule.vcxproj]`

and

`C:\Users\source\repos\projects\ExampleApp\node_modules\react-native-winrt\windows\WinRTTurboModule\Generated Files\rnwinrt\base.h(3034,113): error C3878: syntax error: unexpected token '<' following 'simple-type-specifier' [C:\Users\source\repos\projects\ExampleApp\node_modules\react-native-winrt\windows\WinRTTurboModule\WinRTTurboModule.vcxproj]`

### Question
What are the requirements for react-native-winrt to build properly?

--- a/rnwinrt/rnwinrt/react/strings/base.h
+++ b/rnwinrt/rnwinrt/react/strings/base.h
@@ -3031,7 +3031,7 @@ namespace rnwinrt
                     auto arg0 = convert_value_to_native<winrt::Windows::Foundation::Collections::IObservableVector<T>>(
                         runtime, args[0]);
                     auto arg1 =
-                        convert_value_to_native<winrt::Windows::Foundation::Collections::IVectorChangedEventArgs<T>>(
+                        convert_value_to_native<winrt::Windows::Foundation::Collections::IVectorChangedEventArgs>(
                             runtime, args[1]);
                     value(arg0, arg1);
                     return jsi::Value::undefined();

